api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -46,6 +47,14 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        if (Context.getPersonService().getPersonAttributeTypeByName("accessControlLocation") == null) {
```

Comment:
The code is more readable if you used constants for these.

Reply:
I will change those. Thanks @dkayiwa